.travis/test_prepare.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -ex
+
+SCRIPT=`realpath -s $0`
+SCRIPTPATH=`dirname $SCRIPT`
+
+. $SCRIPTPATH/test_common.sh
+
+(
+echo "building ovn is ${ovn_srcdir}"
```

Comment:
typo: s/is/in

Reply:
Done

Reply:
Sorry, I didn't see it fixed in this revision. Will you send another revision?

Reply:
Sorry. I thought I'd fixed that one. I'll send another revision